### ISSUE TITLE
Scheduler Timeline View - Ability to Freeze row for vertical scrolling

### ISSUE THREAD

**Issue by Efactor888** [fullcalendar/fullcalendar-scheduler#386]
Ability to freeze a row for vertical scrolling in the scheduler's timeline view. This way if there are a ton of rows of data, the user can refer to a specific row of data.

![freezerow](https://user-images.githubusercontent.com/7477196/33461885-4c6efb28-d5e9-11e7-8302-45e33ddeb01a.png)
![freezerow](https://user-images.githubusercontent.com/7477196/33461900-588934c8-d5e9-11e7-902c-e8d4c4799043.png)

**Comment by acerix** [comment]
Thanks for the suggestion.

So if I understand, you would want new functions to freeze/unfreeze a row which would put that row at the top in a fixed position below the header, and it would stay there while you scroll through other events?

**Comment by Efactor888** [comment]
Absolutely! 

Reason for this request is for situations where the user wants to compare schedules of one resource to all others.

Thank you.

**Comment by Efactor888** [comment]
Also, quick question, will it be possible to hardcode this to just the first resource row for the time being? For example, with the tbody css as a function in resource render?

Thank you.

**Comment by acerix** [comment]
Do you mean having one special "frozen" row instead, since that would be quicker to implement than dynamic freezing of rows?

**Comment by Efactor888** [comment]
Yep! So the first resource row in the timeline view will be frozen.

**Comment by bukai0211** [comment]
Hi,
I was looking for the same option for one of my clients. What I need is freeze the resource group. Let me know if i'm on the same page as you guys are.

Thank you very much.
